src/pages/ImportData.js

```diff
@@ -109,36 +134,92 @@ const ImportData = () => {
                 value: selectedWorkspaceId,
                 onChange: setSelectedWorkspaceId
               }),
+              div({ style: { marginTop: '0.5rem', lineHeight: '1.5' } },
+                [noteMessage]),
               div({ style: { display: 'flex', alignItems: 'center', marginTop: '1rem' } }, [
+                h(ButtonSecondary, { onClick: () => setMode(), style: { marginLeft: 'auto' } }, ['Back']),
                 h(ButtonPrimary, {
+                  style: { marginLeft: '2rem' },
                   disabled: !selectedWorkspace,
                   onClick: () => onImport(selectedWorkspace.workspace)
-                }, ['Import']),
-                h(ButtonSecondary, { style: { marginLeft: '1rem' }, onClick: () => setMode() }, ['Back'])
+                }, ['Import'])
+              ])
+            ])
+          }],
+          ['template', () => {
+            return h(Fragment, [
+              div({ style: styles.title }, ['Start with a template']),
+              div({ style: { marginBottom: '1rem', lineHeight: '1.5' } }, [
+                noteMessage
+              ]),
+              div({ style: { overflow: 'auto', maxHeight: '25rem' } }, [
+                _.map(([i, ws]) => {
+                  const { name, namespace, description, hasNotebooks, hasWorkflows } = ws
+                  const isSelected = _.isEqual({ name, namespace }, selectedTemplateWorkspaceKey)
+
+                  return div({
+                    key: `${name}/${namespace}`,
+                    style: {
+                      display: 'flex', alignItems: 'baseline',
+                      marginBottom: '1rem', paddingLeft: '0.25rem',
+                      ...(i > 0 ? { borderTop: Style.standardLine, paddingTop: '1rem' } : {})
+                    }
+                  }, [
+                    h(RadioButton, {
+                      name: 'select-template',
+                      checked: isSelected,
+                      onChange: () => setSelectedTemplateWorkspaceKey({ namespace, name }),
+                      text: h(Collapse, {
+                        buttonStyle: { color: colors.dark(), fontWeight: 600 },
+                        style: { fontSize: 14, marginLeft: '0.5rem' },
+                        title: h(Fragment, [
+                          name,
+                          hasNotebooks && img({ src: jupyterLogo, style: { height: 23, width: 23, marginLeft: '0.5rem' } }),
+                          hasWorkflows &&
+                            wdlIcon({ style: { height: 23, width: 23, marginLeft: '0.5rem', borderRadius: 3, padding: '8px 4px 7px 4px' } })
+                        ])
+                      }, [p({ style: { fontSize: 14, lineHeight: '1.5', marginRight: '1rem' } }, description)])
```

Comment:
nit: It shouldn't be a problem here, but we should bracket children for consistency.
```suggestion
                      }, [p({ style: { fontSize: 14, lineHeight: '1.5', marginRight: '1rem' } }, [description])])
```